app/views/declaration/additionalDocuments/additional_document_edit.scala.html

```diff
@@ -40,6 +40,7 @@
     tariffExpander: tariffExpander,
     saveButtons: saveButtons,
     link: link,
+    additionalDocument: AD,
```

Comment:
Nit: the pseudonym "AD" looks a little odd, maybe the helper class could be renamed "AdditionalDocHelper" to avoid this?